plugins/inputs/aliyuncms/discovery.go

```diff
@@ -65,10 +42,17 @@ type discoveryTool struct {
 	wg       sync.WaitGroup              //WG for primary discovery goroutine
 	interval time.Duration               //Discovery interval
 	done     chan bool                   //Done channel to stop primary discovery goroutine
-	dataChan chan map[string]interface{} //Discovery data
+	DataChan chan map[string]interface{} //Discovery data
```

Comment:
Why do you need to export it?

Reply:
Please have a look on my comment regarding the `discoveryTool` object

Reply:
I will switch it to non-exported